pkg/frontend/installversions_list.go

```diff
@@ -0,0 +1,31 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+func (f *frontend) listInstallableVersions(w http.ResponseWriter, r *http.Request) {
+	log := r.Context().Value(middleware.ContextKeyLog).(*logrus.Entry)
+	versions := f.getInstallVersions()
+	b, err := json.Marshal(versions)
+	reply(log, w, nil, b, err)
+}
+
+// Creating this separate method so that it can be reused while doing some other stuff like validating the install version against available version.
+func (f *frontend) getInstallVersions() []string {
+	// TODO: Currently, the versions are hard coded and being pulled from version.UpgradeStreams, but in future they will be pulled from cosmosdb.
```

Comment:
I would rather see you rebase your PR off https://github.com/Azure/ARO-RP/pull/2165/ and begin to add in the logic to query the OCP version document

Reply:
For this PR, we should keep it as the hardcoded one. There's a lot of work that needs to happen before we can use the data from 2165.

Reply:
So what should I do now .. I have written to the code to fetch it from cosmosdb .. I can write unit tests also .. but I don't think I can manually try it in my dev rp as afaik there are no openshiftversiondocs in cosmosdb.

Reply:
I think for this specific PR, we should keep it like this. Once #2165 and https://github.com/Azure/ARO-RP/pull/2205 land, then we have the capacity to test the cosmosdb interactions.